### PR TITLE
Added multiArchiveTag to startArchive

### PR DESCRIPTION
**Pull request by manchuck**
#### What is this PR doing?

This updates `startArchive` to accept in the `multiArchiveTag` parameter.

#### How should this be manually tested?
```js
opentok.startArchive(
    sessionId, 
    {
        name: 'My awesome session',
        multiArchiveTag: 'tagging this session',
    },
    (err, archive) => {
        if (err) {
            console.err(err);
            return;
        }

        console.log(archive);
);
```

#### What are the relevant tickets?

[DEVX-6489](https://jira.vonage.com/browse/DEVX-6489)

### CODE REVIEW

**Review by jeffswartz — CHANGES_REQUESTED** [review]
Some things are missing:

* Add docs comments for the `multiArchiveTag` option of the `OpenTok.startArchive()` method.
* Add `mutliArchiveTag` as a property of the `Archive` class (and add its docs).
* Implement the `multiArchiveTag` option of the `OpenTok.startBroadcast()` method and the `multiArchiveTag` property of the Broadcast object (and add docs for both).
* This PR has merge conflicts.
* Should this PR target the dev branch instead of main. Also, we need to rev the version (to 2.15.0, I think).

**Comment by manchuck** [comment]
@jeffswartz , I was unable to find the merge conflict. Could it be that changing the base resolved it?

`multiBroadcastTag` was not part of the ticket. @pardel is that part of a different ticket? 

@jeffswartz I'm not sure where to add the property to the class. I did check for this first and did not see a need to add it since the code is composing the class props from the properties passed in:

```js
function Archive(config, properties) {
  var hasProp = {}.hasOwnProperty;
  var id = properties.id;
  var key;

  for (key in properties) {
    if (hasProp.call(properties, key)) {
      this[key] = properties[key];
    }
  }
  // ... snip
}
```

I did add the property to the docs.

**Comment by manchuck** [comment]
Also holding off on bumping the rev since I have another PR waiting to merge.

**Review by hamzanasir — APPROVED** [review]
As long as the documentation for this tag is looked out for this PR looks good to me!

File: lib/archiving.js [hunk 1]
```diff
@@ -114,6 +114,11 @@ var generateHeaders = function generateHeaders(config) {
 *   the URL for 10 minutes. To generate a new URL, call the
 *   {@link OpenTok#getArchive OpenTok.getArchive()} or
 *   {@link OpenTok#listArchives OpenTok.listArchives()} method.
+* 
+* @property {String} multiArchiveTag
+*   The tag of the archive, with which to start. This is unique for each archive within session. 
+*   <code>multiArchiveTag</code> is not case-sensitive (e.g. 'MyArchive' and 'myarchive' are 
+*   treated as same)
```
Comment (jeffswartz) [inline lib/archiving.js:121]: This is not quite accurate. See the docs for `multiArchiveTag` in the REST API docs: https://tokbox.com/developer/rest/#start_archive.

File: lib/opentok.js [hunk 2]
```diff
@@ -199,6 +199,11 @@ OpenTok = function (apiKey, apiSecret, env) {
   *          {@link OpenTok#removeArchiveStream OpenTok.removeArchiveStream()} methods.</li>
   *       </ul>
   *   </li>
+  *   <li>
+  *     <code>multiArchiveTag</code> (String) &mdash; The tag of the archive, with which to start, 
+  *     this is unique for each archive within session. <code>multiArchiveTag</code> is not 
+  *     case-sensitive (e.g. 'MyArchive' and 'myarchive' are treated as same)
+  *   </li>
```
Comment (jeffswartz) [inline lib/opentok.js:206]: This is not quite accurate. See the docs for `multiArchiveTag` in the REST API docs: https://tokbox.com/developer/rest/#start_archive.

**Review by jeffswartz — CHANGES_REQUESTED** [review]
Will we also be adding the `multiBroadcastTag` for start broadcast?

File: lib/archiving.js (hunk 1 above)
Comment (manchuck) [reply lib/archiving.js]: My mistake. I took from https://github.com/opentok/api-review/tree/master/archiving-api

**Comment by manchuck** [comment]
@jeffswartz I also added `multiBroadcastTag` as well

**Review by jeffswartz — APPROVED** [review]

**Pull request merged**